versions.yaml

```diff
@@ -20,6 +20,7 @@ module-sets:
     modules:
       - go.opentelemetry.io/contrib/bridges/prometheus
       - go.opentelemetry.io/contrib/detectors/aws/lambda
+      - go.opentelemetry.io/contrib/detectors/azure/azurevm
```

Comment:
Do we want the first version release of this to be v0.53.0 or should this be its own modset and release it as v0.0.1?

Reply:
Either way is fine for me.

It seems that experimental groups are grouped by topic now (instrumentation, samplers, config, bridge, and processors), each with different versions.

Would you suggesting adding a new group, with version v0.0.1?

Reply:
@MrAlias Done, please have another look.